build-release-tools/docker_build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash 
 set -e
-
+set -x
```

Comment:
I don't think set -x everywhere. which make the console output very hard to debug/read .
Can we use "set -x" only when necessary ?